src/components/dock/dock-button/dock-button.scss

```diff
@@ -44,6 +41,11 @@
         );
         box-shadow: var(--button-shadow-inset);
 
+        &:focus-visible {
+            outline: none;
+            box-shadow: var(--shadow-depth-8-focused);
+        }
+
```

Comment:
could we replace it with @mixins as well?

Reply:
ah it's further up. this is a repetition of the same code 😄 good find

Reply:
Also I noticed that we didn't have `disabled` on dock-button, so I removed the styles for it

Reply:
Good catch anyways!